src/code/z_map_exp.c

```diff
@@ -405,10 +405,14 @@ void Minimap_Draw(GlobalContext* globalCtx) {
                     osSyncPrintf("Game_play_demo_mode_check=%d\n", Gameplay_InCsMode(globalCtx));
                     // clang-format off
                     if (!R_MINIMAP_DISABLED) { Audio_PlaySoundGeneral(NA_SE_SY_CAMERA_ZOOM_UP, &gSfxDefaultPos, 4,
-                                                                      &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb); }
-                    else { Audio_PlaySoundGeneral(NA_SE_SY_CAMERA_ZOOM_DOWN, &gSfxDefaultPos, 4,
-                                                  &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb); }
-                    // clang-format on
+                                                                      &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale,
+                                                                      &gSfxDefaultReverb);
+                        // clang-format on
+                    } else {
+                        Audio_PlaySoundGeneral(NA_SE_SY_CAMERA_ZOOM_DOWN, &gSfxDefaultPos, 4,
+                                               &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale,
+                                               &gSfxDefaultReverb);
+                    }
```

Comment:
Yeah I'm not too picky about reverting a bunch of these changes. I just wanted to try and show how much we can reduce the amount of `clang-format off` blocks.
I feel like even if it looks bad here, it's still a better idea than more code wrongly formatted due to being in a format-off block

You or anyone only have one word to say for me to actually revert these though, as I said I'm not particularly attached to my suggestion

Reply:
I guess I would like to hear more opinions on this, since it seems I wrote all the blocks you broke up like this, and I don't think it was ever discussed in the past

Reply:
I'm with @Roman971 on this one, I remember thinking it at the time, but thought it was more important to mention stuff like assignment in the loops first, which were more "bad code" than "bad style".